src/pages/[id].page.jsx

```diff
@@ -13,9 +13,12 @@ export default function Details() {
   );
 }
 
+export const getStaticPaths = async () => ({
+  paths: [], // indicates that no page needs be created at build time
+  fallback: 'blocking', // indicates the type of fallback
+});
+
 // pre-render this page on each request using data in props.
-export async function getServerSideProps() {
-  return {
-    props: {},
-  };
-}
+export const getStaticProps = async () => ({
+  props: {},
+});
```

Comment:
Does it break anything if this function is omitted? It doesn't seem to be adding much value yet (we will need it for Airtable fetching soon, though).

Reply:
Yes it does break. If you go to [http://localhost:3000/any-text-here](http://localhost:3000/any-text-here) with getStaticProps removed, then it will throw an error because getStaticPaths needs getStaticProps. So if you then also remove getStaticPaths it throws another error.